charts/shoot-core/components/charts/metrics-server/templates/deployment.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   name: metrics-server
   namespace: kube-system
   labels:
+    networking.gardener.cloud/from-seed: allowed
+    networking.gardener.cloud/to-apiserver: allowed
+    networking.gardener.cloud/to-dns: allowed
+    networking.gardener.cloud/to-kubelet: allowed
```

Comment:
All those labels must be add to the Pod template several lines down. This applies for all resources.

Reply:
done, thanks

Reply:
They are not needed in `Deployment.metadata.labels` they must only be in `Deployment.spec.template.metadata.labels`. Remove them here.